docs/components/inputMapper.base.md

```diff
@@ -0,0 +1,300 @@
+- Returns: Nothing.
+
+Listens for the changes in gamepad's input values stored as `inputMapper.base` component's model data and calls the
+**navigation-producing invokers** according to the configuration map. The navigation-producing invoker is triggered only
+if the component provides it. Each of these invokers is passed with the following arguments:
+
+- `inputValue`: Current value of the gamepad input.
+- `speedFactor`: Times by which the speed of given (continuous) action should be increased.
+- `invert`: Whether direction of the given action should be in the opposite order (for thumbsticks).
+- `background`: Whether a new tab or window should open in background.
+- `oldInputValue`: Previous value of the gamepad input before its state is changed.
+- `homepageURL`: URL that a new tab or window should load when opened.
+
+The invokers use only those arguments required to perform the navigation it's meant for. Apart from the `inputValue` and
+`oldInputValue` arguments, the remaining arguments derive their value from the configuration map.
+
+### `{inputMapper.base}.clearIntervalRecords(records)`
+
+- `records {Object}` The `intervalRecords` member containing a list of interval loops.
+- Returns: Nothing.
+
+Clears all the running interval loops when the gamepad is disconnected or when the instance of the `inputMapper.base`
+component is destroyed.
+
+### `{inputMapper.base}.tabindexSortFilter(elementOne, elementTwo)`
+
+- `elementOne {Object}` The DOM element to be compared and sorted.
+- `elementTwo {Object}` The DOM element to be compared and sorted.
+- Returns: `Integer` An integer determining the order of two elements.
+
+Sorts all the tabbable elements in the order of their `tabindex` value. If none of the elements have a `tabindex`
+attribute then the elements are placed in the given order.
+
+### `{inputMapper.base}.trackDOM(that)`
+
+- `that {Object}` The `inputMapper.base` component.
+- Returns: Nothing.
+
+Listens for changes to the DOM using `MutationObserver`. When the DOM has loaded or is changed, it creates a list of
+tabbable elements, sorts them using
+[`{inputMapper.base}.tabindexSortFilter`](#inputmapperbasetabindexsortfilterelementone-elementtwo), and then stores them
+the `tabbableElements` member variable for use during tab navigation.
+
+### `{inputMapper.base}.stopTrackingDOM()`
+
+- Returns: Nothing.
+
+Stops the `MutationObserver` called by the [`{inputMapper.base}.trackDOM`](#inputmapperbasetrackdomthat) when the
+instance of the `inputMapper.base` component is destroyed. It avoids listening changes to the DOM any further.
+
+## Navigation Invokers
+
+### `{inputMapper.base}.click(value)`
+
+- `value {Number}` Current value of the gamepad input.
+- Returns: Nothing.
+
+Performs **click** on the focused element when the gamepad `button` or `trigger` is pressed. However, the `select`
+dropdown is toggled by changing its `size` attribute when clicked using the gamepad.
```

Comment:
Given that everything the extension does uses the gamepad, you can leave that phrase out.